docs/project-setup/existing-projects.md

```diff
@@ -1,3 +1,55 @@
 # Existing Projects
 
-To Be Done
\ No newline at end of file
+To add Outrigger support to an existing project, you'll need to understand how to create a Docker Compose and an 
+`.outrigger.yml` file, how to configure DNS for your project's containers and how to configure volumes and persistent 
```

Comment:
Should we call out that .outrigger.yml is optional?

This seems to form a good list of top level headings that the rest of the doc should be structured around. There is already a Docker Compose heading but the others get a little lost. I've dropped in some notes below to highlight where they might be added. One augmentation might be to change the volumes and persistent data storage note to something like "how to share code and configure persistent data storage" and add a small section introducing that volumes and unison and the /data directory are the things they need to know about to do that with pointers to the relevant deeper documentation pages.